Dockerfile

```diff
@@ -9,32 +9,39 @@ ARG DEBIAN_FRONTEND
 RUN dpkg --add-architecture armel \
     && dpkg --add-architecture armhf \
     && dpkg --add-architecture arm64 \
-    && dpkg --add-architecture ppc64el \
-    && apt-get update && apt-get install -y \
-    build-essential \
-    curl \
-    sudo \
-    gawk \
-    iptables \
-    jq \
-    pkg-config \
-    libaio-dev \
-    libcap-dev \
-    libprotobuf-dev \
-    libprotobuf-c0-dev \
-    libnl-3-dev \
-    libnet-dev \
-    libseccomp2 \
-    libseccomp-dev \
-    protobuf-c-compiler \
-    protobuf-compiler \
-    python-minimal \
-    uidmap \
-    kmod \
-    crossbuild-essential-armel crossbuild-essential-armhf crossbuild-essential-arm64 crossbuild-essential-ppc64el \
-    libseccomp-dev:armel libseccomp-dev:armhf libseccomp-dev:arm64 libseccomp-dev:ppc64el \
-    --no-install-recommends \
-    && apt-get clean
+    && dpkg --add-architecture ppc64el
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
What is the reason for splitting a single RUN statement into two?

Reply:
The `dpkg --add-architecture` does not have to be re-run when we do `apt-get update`, and I thought it was slightly cleaner, but let me know if you feel strongly about that

Reply:
I like a single `RUN` slightly better, but I'm fine either way